deploy/helm/templates/statefulset.yaml

```
@@ -74,6 +74,11 @@ spec:
               containerPort: 662
               protocol: UDP
           securityContext:
+            {{- range $key, $value := .Values.extraArgs }}
+            {{- if and (eq $key "enable-xfs-quota") $value }}
+            privileged: true
+            {{- end }}
+            {{- end }}
             capabilities:
               add:
                 - DAC_READ_SEARCH
```

I'll suggest the same I did in #32. The cap SYS_ADMIN should be enough to enable xfs quota feature. @consideRatio confirmed that https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/pull/38#issuecomment-902653475

```suggestion
            capabilities:
              add:
                {{- range $key, $value := .Values.extraArgs }}
                {{- if and (eq $key "enable-xfs-quota") $value }}
                - SYS_ADMIN
                {{- end }}
                {{- end }}
                - DAC_READ_SEARCH
```